app/src/main/res/layout-land/activity_movie_details.xml

```diff
@@ -17,14 +17,13 @@
 
             <ImageView
                 android:id="@+id/iv_header"
-                android:layout_width="745dp"
-                android:layout_height="686dp"
+                android:layout_width="850dp"
```

Comment:
Try to avoid concrete values if possible use march_parent or relative size by constrained with other views.